packages/util-retry/src/defaultRetryToken.ts

```diff
@@ -1,91 +1,53 @@
-import { RetryErrorInfo, RetryErrorType, StandardRetryBackoffStrategy, StandardRetryToken } from "@aws-sdk/types";
+import { RetryErrorInfo, RetryErrorType, StandardRetryToken } from "@aws-sdk/types";
 
 import {
-  DEFAULT_RETRY_DELAY_BASE,
+  INITIAL_RETRY_TOKENS,
   MAXIMUM_RETRY_DELAY,
   NO_RETRY_INCREMENT,
   RETRY_COST,
-  THROTTLING_RETRY_DELAY_BASE,
   TIMEOUT_RETRY_COST,
 } from "./constants";
-import { getDefaultRetryBackoffStrategy } from "./defaultRetryBackoffStrategy";
-
-/**
- * @public
```

Comment:
this is marked public but is only used as the input type of an internal function, so I suggest we remove it